### DISCUSSION TITLE
TheHive API not working

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#4284] [Unsupported Versions]
Hello,

My Hive API seems to not be working. I am running the command below locally from thehive server, but it fails. I am able to escalate to thehive from the Kibana console, but I am looking to use this for some other external alarm integrations.

Command:
curl -XPOST -H 'Authorization: Bearer myapikey' -H 'Content-Type: application/json' http://localhost:9000/api/case -d '{
  "title": "My first case",
  "description": "This case has been created by my custom script"
}'

Error: A client error occurred on POST /api/case

**Comment by weslambert** [comment]
If running locally, try:

```
curl -XPOST -H 'Authorization: Bearer myapikey' -H 'Content-Type: application/json' http://localhost:9000/thehive/api/case -d '{
"title": "My first case",
"description": "This case has been created by my custom script"
}'
```

**Reply by glwallum** [reply]
Thanks Wes, that worked

If we wanted to do remote API to the hive, would this work the same way? It looks like thehive api is only accessible locally, would we need to change some configuration?

**Reply by weslambert** [reply]
The POST would need to be to:

`https://SECURITYONION/thehive/api/case`
